xobjects/ref.py

```diff
@@ -297,9 +297,10 @@ def _get_inner_types(cls):
 
     def _to_json(self):
         v = self.get()
+        w = v
```

Comment:
what about `classname=v.__class__.__name__`?  instead of `w=v`  which is a bit obscure